<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QDialog" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>630</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>326</width>
     <height>580</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>321</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="FilterGroupLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filers Setup</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="HueVals">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>121</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Hue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QSlider" name="maxVS_H">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>47</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="minSB_H">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="minVS_H">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxSB_H">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SatVals">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>121</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Saturation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QSlider" name="maxVS_S">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>47</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="minSB_S">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="minVS_S">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxSB_S">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ValVals">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>121</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Value</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QSlider" name="maxVS_V">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>47</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="minSB_V">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="minVS_V">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxSB_V">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Results</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="ErodeBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>490</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Erode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QSpinBox" name="SizeSB_E">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="RepsSB_E">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="GeometryCB_E">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cross</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="DecreaseCHB_E">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Decrease                      </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Geometry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DilateBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>490</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Dilate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QSpinBox" name="SizeSB_D">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="RepsSB_D">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>47</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="GeometryCB_D">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cross</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="DecreaseCHB_D">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Decrease                      </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Geometry</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="FiltersList">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilePathLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadFileButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>620</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibrationWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibrationWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
